docs/dev/guides/backend-routes.md

```diff
@@ -26,27 +26,39 @@ and is configured through annotations.
 // src/Controller/BackendController.php
 namespace App\Controller;
 
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment as TwigEnvironment;
 
 /**
- * @Route("/contao", defaults={
- *     "_scope" = "backend",
- *     "_token_check" = true,
- *     "_backend_module" = "my-modules"
- * })
+ * @Route("/contao//my-backend-route",
+ *     name="app.backend-route" 
```

Comment:
```suggestion
 *     name=App\Action\BackendAction::class,
```

Reply:
I have never seen a route with a class name, not sure this works (and is best practice?)

Reply:
Yes this works and is much more useful as you can then simply write

```
$this->router->generate(BackendAction::class)
```

in PHP.

Reply:
There is a trailing comma missing after `name=""` which leads to `AnnotationException.php line 42: [Syntax Error] Expected Doctrine\Common\Annotations\DocLexer::T_CLOSE_PARENTHESIS, got 'defaults'` error